.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,3 +16,20 @@ Bullet points like
 really help with making this more readable.
 
 Fixes/Closes/Related-to #1 (enter issue number, except in rare cases where none exists).
+
+It is also helpful to add an emoji before the commit message to identify the kind of work done on a single commit. See the following suggestions below:
+
+- :bug: - (bug) bug fix
+- :sparkles: - (sparkles) new feature
+- :exclamation: - (exclamation) tests
+- :fire: - (fire) P0
+- :rocket: - (rocket) performance improvements
+- :lipstick: - (lipstick) css/styling
+- :wheelchair: - (wheelchair) accessability
+- :book: - (book) documentation
+- :recycle: - (recycle) refactoring (like moving around code w/o any changes)
+- :building_construction: - (building_construction) infrastructure and tooling
+- :rewind: - (rewind) revert
+- :feelsgood: - (feelsgood) submitting a hacky PR
+- :construction_worker: - (construction_worker) builds/travis related
+- :put_litter_in_its_place: - (put_litter_in_its_place) removing code
```

Comment:
Maybe say "code cleanup" instead of "removing code"?

Reply:
code cleanup is even more ambiguous though right? like a cleanup could be moving code around like refactoring etc.

Reply:
Fair enough :)